[2024-07-10 01:09:06,959] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: create_layers_in_snowflake.check_seeds manual__2024-07-10T01:09:06.507425+00:00 [queued]>
[2024-07-10 01:09:06,970] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: create_layers_in_snowflake.check_seeds manual__2024-07-10T01:09:06.507425+00:00 [queued]>
[2024-07-10 01:09:06,970] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 01:09:06,971] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-07-10 01:09:06,971] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 01:09:06,984] {taskinstance.py:1377} INFO - Executing <Task(DockerOperator): check_seeds> on 2024-07-10 01:09:06.507425+00:00
[2024-07-10 01:09:06,988] {standard_task_runner.py:52} INFO - Started process 1992 to run task
[2024-07-10 01:09:06,992] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'create_layers_in_snowflake', 'check_seeds', 'manual__2024-07-10T01:09:06.507425+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/create_layers.py', '--cfg-path', '/tmp/tmpcy7r_xzx', '--error-file', '/tmp/tmpdkllg2kf']
[2024-07-10 01:09:06,992] {standard_task_runner.py:80} INFO - Job 16: Subtask check_seeds
[2024-07-10 01:09:07,049] {task_command.py:369} INFO - Running <TaskInstance: create_layers_in_snowflake.check_seeds manual__2024-07-10T01:09:06.507425+00:00 [running]> on host 9ba3cd856d5b
[2024-07-10 01:09:07,121] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_layers_in_snowflake
AIRFLOW_CTX_TASK_ID=check_seeds
AIRFLOW_CTX_EXECUTION_DATE=2024-07-10T01:09:06.507425+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-10T01:09:06.507425+00:00
[2024-07-10 01:09:07,142] {docker.py:247} INFO - Starting docker container from image custom_dbt_image
[2024-07-10 01:09:08,959] {docker.py:308} INFO - [0m01:09:08  Running with dbt=1.5.0
[2024-07-10 01:09:08,960] {docker.py:308} INFO - [0m01:09:08  dbt version: 1.5.0
[2024-07-10 01:09:08,961] {docker.py:308} INFO - [0m01:09:08  python version: 3.7.3
[2024-07-10 01:09:08,961] {docker.py:308} INFO - [0m01:09:08  python path: /usr/bin/python3
[2024-07-10 01:09:08,962] {docker.py:308} INFO - [0m01:09:08  os info: Linux-6.6.31-linuxkit-x86_64-with-debian-10.13
[2024-07-10 01:09:08,962] {docker.py:308} INFO - [0m01:09:08  Using profiles.yml file at /usr/app/profiles.yml
[2024-07-10 01:09:08,963] {docker.py:308} INFO - [0m01:09:08  Using dbt_project.yml file at /usr/app/dbt_project.yml
[2024-07-10 01:09:08,963] {docker.py:308} INFO - [0m01:09:08  Configuration:
[2024-07-10 01:09:09,203] {docker.py:308} INFO - [0m01:09:09    profiles.yml file [[32mOK found and valid[0m]
[2024-07-10 01:09:09,231] {docker.py:308} INFO - [0m01:09:09    dbt_project.yml file [[32mOK found and valid[0m]
[2024-07-10 01:09:09,232] {docker.py:308} INFO - [0m01:09:09  Required dependencies:
[2024-07-10 01:09:09,238] {docker.py:308} INFO - [0m01:09:09   - git [[32mOK found[0m]
[2024-07-10 01:09:09,239] {docker.py:308} INFO - [0m01:09:09  Connection:
[2024-07-10 01:09:09,239] {docker.py:308} INFO - [0m01:09:09    account: tsb24868.us-east-1
[2024-07-10 01:09:09,240] {docker.py:308} INFO - [0m01:09:09    user: dbt
[2024-07-10 01:09:09,240] {docker.py:308} INFO - [0m01:09:09    database: AIRBNB
[2024-07-10 01:09:09,241] {docker.py:308} INFO - [0m01:09:09    schema: DEV
[2024-07-10 01:09:09,241] {docker.py:308} INFO - [0m01:09:09    warehouse: COMPUTE_WH
[2024-07-10 01:09:09,242] {docker.py:308} INFO - [0m01:09:09    role: transform
[2024-07-10 01:09:09,242] {docker.py:308} INFO - [0m01:09:09    client_session_keep_alive: False
[2024-07-10 01:09:09,242] {docker.py:308} INFO - [0m01:09:09    query_tag: None
[2024-07-10 01:09:10,448] {docker.py:308} INFO - [0m01:09:10    Connection test: [[32mOK connection ok[0m]
[2024-07-10 01:09:10,449] {docker.py:308} INFO - [0m01:09:10  [32mAll checks passed![0m
[2024-07-10 01:09:11,024] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=create_layers_in_snowflake, task_id=check_seeds, execution_date=20240710T010906, start_date=20240710T010906, end_date=20240710T010911
[2024-07-10 01:09:11,079] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-07-10 01:09:11,112] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
